lib/l10n_utils/tests/test_files/templates/404-locale.html

```diff
@@ -0,0 +1,6 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
```

Comment:
I'm assuming this file just needs to exist to make the tests work happily in isolation, yep?

Reply:
yep!